model/src/main/java/de/danoeh/antennapod/model/feed/PodcastIndexChapter.java

```diff
@@ -0,0 +1,67 @@
+package de.danoeh.antennapod.model.feed;
+
+import android.util.Log;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PodcastIndexChapter extends Chapter {
+    public static final int PODCASTINDEX_CHAPTER = 4;
+    public static String TAG = "PodcastIndexChapter";
+
+    public PodcastIndexChapter(int startTime,
+                               String title,
+                               String link,
+                               String imageUrl) {
+        super(startTime * 1000, title, link, imageUrl);
+    }
+
+    @Override
+    public int getChapterType() {
+        return PODCASTINDEX_CHAPTER;
+    }
+
+    @Override
+    public String toString() {
+        return "PodcastIndexChapter "
+                + getTitle()
+                + " " + getStart()
+                + " " + getLink()
+                + " " + getImageUrl();
+    }
+
+    public static List<Chapter> parseChapters(String jsonStr) {
```

Comment:
Hmm, the model should only be a data store, not do business logic like parsing. Please move the method to another class. Maybe in the `:parser:feed` module? It's not a perfect match there but starting a new module just for PodcastIndex chapter parsing would be a bit too much